### PR TITLE
Fix glitch when moving from a tab on my profile to default profile view

### PR DESCRIPTION
**Pull request by YLeight**
## Describe your changes
I've remove redundant page refresh after pushing into the history url of default profile view

## Issue ticket number and link
https://github.com/stakwork/sphinx-tribes/issues/1034

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

https://jam.dev/c/6d094992-5dcc-48f3-8147-9b4ebeaa8d49

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
okay the video looks good to me. great work! Let's do a code review.

File: frontend/app/yarn.lock
Comment (kevkevinpal) [inline frontend/app/yarn.lock:1]: why is the yarn.lock updated???

Comment (kevkevinpal) [reply frontend/app/yarn.lock:1]: you can run this to remove the yarn.lock back to master 
`git checkout origin/master -- frontend/app/yarn.lock`

Comment (YLeight) [reply frontend/app/yarn.lock:1]: > why is the yarn.lock updated???

Deps in `package.json` is updated as in registry used packages was updated and their new versions are compatible with version in `package.json` file as they were captured there with `^`

Comment (YLeight) [reply frontend/app/yarn.lock:1]: > you can run this to remove the yarn.lock back to master `git checkout origin/master -- frontend/app/yarn.lock`

I'll remove this commit from PR

Comment (YLeight) [reply frontend/app/yarn.lock:1]: @kevkevinpal done, I have updated this PR
Can I resolve this conversation?

**Pull request merged**